python/pyspark/ml/connect/feature.py

```diff
@@ -256,3 +262,124 @@ def _load_core_model(self, path: str) -> None:
         self.scale_values = sk_model.scale_
         self.mean_values = sk_model.mean_
         self.n_samples_seen = sk_model.n_samples_seen_
+
+
+class VectorAssembler(
+    Transformer,
+    HasInputCols,
+    HasOutputCol,
+    HasInputFeatureSizeList,
+    HasHandleInvalid,
+    ParamsReadWrite,
+):
+    """
+    A feature transformer that merges multiple input columns into a array type column.
```

Comment:
```suggestion
    A feature transformer that merges multiple input columns into a vector type column.
```

Reply:
this assembler output array type :) we don't need to keep exactly the same behavior with old one.

Reply:
What about rename the class to ArrayAssembler ?